tests/kernel_bundle/use_kernel_bundle_invoke_kernel_without_bundle_with_second_queue.cpp

```diff
@@ -0,0 +1,106 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provide verification for invoking kernel that not contained in kernel bundle
+//  the test use secondary queue. The test will be skipped if kernel for
+//  execution will be found in obtained kernel bundle.
+//
+//  The test verifies that the exception with sycl::errc::kernel_not_supported
+//  was thrown.
+//
+*******************************************************************************/
+
+#include "../common/common.h"
+#include "use_kernel_bundle.h"
+
+#define TEST_NAME \
+  use_kernel_bundle_invoke_kernel_without_bundle_with_second_queue
+
+namespace TEST_NAMESPACE {
+using namespace sycl_cts;
+using namespace sycl_cts::tests::use_kernel_bundle;
+
+/** @brief Call use_kernel_bundle and invoke the kernel that not contained in
+ *         provided kernel bundle
+ *  @param log sycl_cts::util::logger class object
+ *  @param ctx Context that will be used for constructing sycl::queue
+ *  @param kernel_bundle empty, non-empty or kernel_bundle with built-in kernels
+ */
+void run_verification(
+    sycl_cts::util::logger &log, const sycl::context &ctx,
+    sycl::kernel_bundle<sycl::bundle_state::executable> &kernel_bundle) {
+  sycl::queue first_queue(ctx, ctx.get_devices()[0]);
+  sycl::queue second_queue(ctx, ctx.get_devices()[0]);
+
+  auto kernel_id = sycl::get_kernel_id<kernel>();
+  if (kernel_bundle.has_kernel(kernel_id)) {
+    log.note(
+        "Test skipped due to kernel bundle contain kernel that shouldn't be in "
+        "this bundle");
+    return;
+  }
+
+  bool ex_was_thrown = false;
+  try {
+    first_queue.submit(
+        [&](sycl::handler &cgh) {
+          cgh.use_kernel_bundle(kernel_bundle);
+          cgh.single_task<kernel>([=]() {});
```

Comment:
```suggestion
          cgh.single_task<kernel>([=] {});
```
all these can be simplified

Reply:
I've applied this comment [here](https://github.com/KhronosGroup/SYCL-CTS/pull/225/commits/e157b8bd72174f5b18b4e387b9dfb4e353573bd7#diff-2e891a7394b6c8dfaf6562b100ea5ecc221d6aec61281549a988fe39eda86283L56-L57)